src/core.c

```diff
@@ -657,46 +658,65 @@ static void VBlankIntr(void)
 struct GraphicsData_Hack {
     uintptr_t src;
     uintptr_t dest;
-    u16 remainingBytes;
+    u16 size;
 };
 
 #define COPY_CHUNK_SIZE 1024
 
 static bool32 ProcessVramGraphicsCopyQueue(void)
 {
     u32 offset;
+#ifndef NON_MATCHING
```

Comment:
So, does it definitely not match if the pointer type is wrong? Is there any reason not to just use the non hack one?